### ISSUE TITLE
Official Rust Gamedev Discord

### ISSUE THREAD

**Issue by AngelOnFira** [rust-gamedev/wg#99]
I've been speaking with @dasifefe about using their Discord server for the monthly community meetup (see #98). The server already has many members that are interested in the game dev side of Rust. It would allow us to get a larger audience for the meetups, and us access to whatever channels we needed for it. Here is the link: [https://discord.gg/yNtPTb2](https://discord.gg/yNtPTb2)

I wanted to get people's ideas on making it the "official" Rust Game Dev discord. Of course, there are lots of places for the community to hang out and chat on Discord already, such as the Rust, Amethyst, Bevy, Embark, and Veloren servers among others. However, this would be a more general location for anyone to be able to go that is specifically for gamedev in Rust.

I've discussed with @dasifefe about their thoughts on this, and the changes that would need to be made. So far, I think the biggest thing is that it would have to have moderators from the WG, which sounds good to @dasifefe. But other items that people think of would be good to discuss 👌 

Of course, there is an argument to be made about people who want to use other tools like 
Mastodon instead. However, I believe this Discord server is already 1000+ members, and so it would seem that this would make a good generic meeting place. And there are options to set up bridges or other methods of linking.

**Comment by AlexEne** [comment]
I think it's fine.

**Comment by aclysma** [comment]
> "making it the "official" Rust Game Dev discord"

I think we should be careful to message this as "A popular place to discuss rust gamedev" rather than "THE place to discuss rust gamedev". In particular some people will not use discord because it is not an open platform. For example, cell phone verification rubs some people the wrong way. This might be splitting hairs, but the WG "choosing" discord as "the" platform could appear exclusionary, but observing/signal-boosting that a discord server is very popular in the community seems nothing but helpful :)

> have to have moderators from the WG

Does this mean anything more than sharing permissions to create/run the voice chat that is hosted by the WG? Just want to make sure we're not talking about the WG taking responsibility for enforcing conduct on a community-run server. :)

**Comment by dasifefe** [comment]
On this server, the only requirement for regular members is to have a verified e-mail. But, because we intend to be in the Server Discovery in the future, moderators require a verified phone.

> Does this mean anything more than sharing permissions to create/run the voice chat that is hosted by the WG? Just want to make sure we're not talking about the WG taking responsibility for enforcing conduct on a community-run server. :)

I'm open to whatever new moderators feel comfortable to do (moderate only the meetings or also moderate the server). The thing I care about is having a nice place for the gamedev community.

**Comment by AngelOnFira** [comment]
@aclysma I covered your first point at the bottom of my post.

> Of course, there is an argument to be made about people who want to use other tools like Mastodon instead. However, I believe this Discord server is already 1000+ members, and so it would seem that this would make a good generic meeting place. And there are options to set up bridges or other methods of linking.

Further, other types of forum discussion would be best done elsewhere, but I think this repo is a pretty good candidate for that. I think that the Discord should be considered "THE" place for chatting, so people know of one place to go. We can then link out to other places from there quite easily, such as important websites, other Discord servers, and so on.

Your second point is more of a clause for us to be able to know that this server will follow our level of moderation. It isn't conscripting us to take the role, however it makes sure that the Rust Gamedev CoC is followed. @dasifefe has been very pleasant on this front; they agree that moderation is important for keeping a clean server. That being said, I plan on being a moderator on it, and handling any issues that may arise. We can also add any more mods from the Rust Gamedev team should we need.

**Comment by darthdeus** [comment]
>  This might be splitting hairs, but the WG "choosing" discord as "the" platform could appear exclusionary, but observing/signal-boosting that a discord server is very popular in the community seems nothing but helpful :)

Personally while I completely agree with this and I dislike closed platforms I also don't use any other chat than Discord unless I absolutely have to. There's just so many these days, and it's beyond annoying when a library/language I like uses some new service that nobody else uses. I know popularity isn't a good argument, but especially in gaming and to a large extent in gamedev, Discord is "the platform".

**Comment by AlexEne** [comment]
@aclysma  I see how I wrote could be missinterpreted, I meant making it the "official" place to host meetings (now it's just mentioned in this issue here). 
Before, the "official" place for meetings was hangouts, so not that much open anyway. 

In any case, there are a bunch of places mentioned on that readme I linked (including reddit, and a room in the rust discord community server), we should also add this server and mention it that meetings are hosted there from now on.  Maybe we could also mention also mention the embark discord one if some embark people are ok with that.

**Comment by 17cupsofcoffee** [comment]
> In any case, there are a bunch of places mentioned on that readme I linked (including reddit, and a room in the rust discord community server), we should also add this server and mention it that meetings are hosted there from now on. Maybe we could also mention also mention the embark discord one if some embark people are ok with that.

This is the most useful/actionable thing IMO. We don't necessarily have to make any big decisions about what the 'one true' meeting place is - for me, the main value of partnering with the Discord would be that we could direct people there without having to add on a disclaimer about it not being run/moderated by the working group.

**Comment by aclysma** [comment]
1) I'm in favor of doing our meetings on discord, and doing it on the suggested server sounds fine to me.
2) I just want to make sure we don't play favorites in promoting this discord server over others, or message it as "THE" place for "rust game development". The original post was "I wanted to get people's ideas on making it the "official" Rust Game Dev discord" - the way that was worded to me implied promoting it exclusively, and for more than a particular meeting.
3) This is not at all meant negatively towards this discord server or people running it. I enjoy being part of it and think it's been well-run. This is coming from my own take on the spirit of our charter - https://github.com/rust-gamedev/wg/blob/master/charter.md - while it doesn't deal with "promoting communities" explicitly, I do think the last bit in spirit can be read "Our purpose is to help such communities to exist, not to promote one community over another.").
4) I don't think the WG **as an organization** should take responsibility for moderation anywhere but this GitHub or our channel within the *actual* official rust discord: https://discord.gg/NkRK9nZ5 - I think it should be done on an individual basis.

**Comment by logicsoup** [comment]
I agree with @aclysma. It makes sense to host meetings on the Rust Gamedev Discord, but making it an “official
“ space for Rust gamedev would be a bit exclusionary. That would also make the WG sort-of responsible for everything happening in that community, which is a lot of additional responsibility. We could use it for meetings as long as the CoC/respectful community guidelines are followed in that space. If any unsolvable problems arise in the future (let’s hope that won’t ever happen) it will be much easier to switch platforms from an “unofficial” space than an “official” one.

**Comment by floer32** [comment]
This can be closed, yeah?

For posterity:

https://github.com/rust-gamedev/wg#join-the-fun
